pkg/costmodel/router.go

```diff
@@ -606,6 +607,8 @@ func Initialize(router *httprouter.Router, additionalConfigWatchers ...*watcher.
 	router.POST("/serviceKey", a.AddServiceKey)
 	router.GET("/helmValues", a.GetHelmValues)
 
+	InitializeQuota(router, kubeClientset)
```

Comment:
We shouldn't be using the `kubeClientset` directly for anything really -- use the `k8sCache` (`clustercache.ClusterCache`) instead.